internal/kibana/ingestmanager/client_agents.go

```diff
@@ -54,6 +54,7 @@ func (c *Client) AssignPolicyToAgent(a Agent, p Policy) error {
 		return fmt.Errorf("could not assign policy to agent; API status code = %d; response body = %s", statusCode, respBody)
 	}
 
+	// TODO make sure the policy has been propagated to agents.
```

Comment:
@ruflin is there any Kibana API that can tell that the policy has been propagated to all agents?

Reply:
Maybe `GET http://localhost:5601/api/ingest_manager/fleet/config/{configId}/agent-status`?

Reply:
@nchaulet ^

Reply:
Yes the config agent status API can give you the number of online agent for a policy 

Otherwise you can probably use the agent search API 
```
GET /api/ingest_manages/fleet/agents?kuery=fleet-agents.policy_revision:your-policy-revision and fleet-agents.polidy_id: your-policy-id
```

Reply:
Thanks!

Reply:
I looked into proposed method. It seems that the `agent-status` will do the trick:

```
GET http://localhost:5601/api/ingest_manager/fleet/agent-status?policyId=6bfd9830-026b-11eb-9819-a1a3afee1cc2

{"results":{"events":5,"total":1,"online":1,"error":0,"offline":0,"other":0}}
```

Reply:
Fixed.